tests/serverpod_test_server/test_integration/ssl/test_certificate.pem

```diff
@@ -0,0 +1,20 @@
+-----BEGIN CERTIFICATE-----
+MIIDOTCCAiGgAwIBAgIUUmZAMo/6EO73Uh8UPTS0pTnUFwYwDQYJKoZIhvcNAQEL
+BQAwRTELMAkGA1UEBhMCQVUxEzARBgNVBAgMClNvbWUtU3RhdGUxITAfBgNVBAoM
+GEludGVybmV0IFdpZGdpdHMgUHR5IEx0ZDAeFw0yNTAxMjgxMjUyMzJaFw0yNjAx
+MjgxMjUyMzJaMEUxCzAJBgNVBAYTAkFVMRMwEQYDVQQIDApTb21lLVN0YXRlMSEw
+HwYDVQQKDBhJbnRlcm5ldCBXaWRnaXRzIFB0eSBMdGQwggEiMA0GCSqGSIb3DQEB
+AQUAA4IBDwAwggEKAoIBAQC1zLj0HJJXYu064zbphsMX1EtGYzTOKkKjuPG4eWKN
+ZLritjanmUcYdvBaEdSayHj9E6/TMrRIcqzomjlpz10WRSgO6tucCJ5nGVLMqnpP
```

Comment:
Suggestion: Put these files in an `asset` folder or similar so we differentiate them from the tests them selfs.